src/goto-programs/remove_static_init_loops.cpp

```diff
@@ -65,23 +70,16 @@ void remove_static_init_loopst::unwind_enum_static(
         size_t unwinds=class_type.get_size_t(ID_java_enum_static_unwind);
 
         unwind_max=std::max(unwind_max, unwinds);
-        if(fname.length()>java_clinit.length())
+        if(unwinds>0)
         {
-          // extend unwindset with unwinds for <clinit> of enum
-          if(fname.compare(
-            fname.length()-java_clinit.length(),
-            java_clinit.length(),
-            java_clinit)==0 && unwinds>0)
-          {
-            const std::string &set=options.get_option("unwindset");
-            std::string newset;
-            if(set!="")
-              newset=",";
-            newset+=
-              id2string(ins.function)+"."+std::to_string(loop_id)+":"+
-              std::to_string(unwinds);
-            options.set_option("unwindset", set+newset);
-          }
+          const std::string &set=options.get_option("unwindset");
+          std::string newset;
+          if(set!="")
+            newset=",";
+          newset+=
+            id2string(ins.function)+"."+std::to_string(loop_id)+":"+
+            std::to_string(unwinds);
+          options.set_option("unwindset", set+newset);
```

Comment:
Can you add a comment in https://github.com/diffblue/cbmc/blob/master/src/cbmc/cbmc_parse_options.cpp#L553 to make this side-effect more visible?